source/draw/gpu/opengl/OpenGLMonochrome.ooc

```diff
@@ -24,12 +24,13 @@ OpenGLMonochrome: class extends OpenGLPacked {
 		this init(rasterImage size, rasterImage stride, rasterImage buffer pointer, context)
 	}
 	toRasterDefault: override func -> RasterImage {
-		result := RasterMonochrome new(this size)
+		stride := ((this size x + 3) / 4) * 4
```

Comment:
Can you create a function for this instead? `alignWidth(this size x, 4)` or something like that